controllers/grid/users/reviewer/ReviewerGridRow.php

```diff
@@ -215,6 +220,31 @@ public function initialize($request, $template = null)
                     )
                 );
             }
+
+            $roleDao = DAORegistry::getDAO('RoleDAO'); /** @var RoleDAO $roleDao */
+            $context = $request->getContext();
+
+            if ($roleDao->userHasRole($context->getId(), $user->getId(), [Role::ROLE_ID_SITE_ADMIN, Role::ROLE_ID_MANAGER]) && $reviewAssignment->getDateConfirmed() == null) {
+                $reviewAssignmentId = $reviewAssignment->getId();
+                $initials = $context->getData('acronym', $context->getPrimaryLocale());
+                $action = "/$initials/api/v1/reviews/$submissionId/$reviewAssignmentId/confirmReview";
```

Comment:
I'd recommend using the `$dispatcher->url()` utility to construct the API URL instead. Here's an example, but it's used throughout the application as well:

```php
$request->getDispatcher()->url($request, PKPApplication::ROUTE_API, $submissionContext->getPath(), 'submissions/' . $this->submission->getId() . '/publications/' . $this->publication->getId());
```